src/app/search/search-filters.ts

```diff
@@ -668,13 +668,13 @@ export function searchFilters(
         return compareByOperand(item.quality.min, predicate);
       },
       hasRating(item: DimItem, predicate: string) {
-        return predicate.length !== 0 && item.dtrRating.overallScore;
+        return item.dtrRating && predicate.length !== 0 && item.dtrRating.overallScore;
```

Comment:
Typo: `dtrRatihg`. You can check `predicate.length` first, too.

Reply:
But there's no typo? I'll swap the order around.